src/rdkafka_msgset_reader.c

```diff
@@ -615,6 +617,7 @@ rd_kafka_msgset_reader_msg_v0_1 (rd_kafka_msgset_reader_t *msetr) {
         /* Enqueue message on temporary queue */
         rd_kafka_q_enq(&msetr->msetr_rkq, rko);
         msetr->msetr_msgcnt++;
+        msetr->msetr_msg_bytes += rkm->rkm_key_len + rkm->rkm_len;
```

Comment:
i did a test in which i produced a single message with a 1 byte key and 1 byte value. i see `"rxmsg_bytes": 3`, which surprised me.

Reply:
Huhm, that is surprising. Is it reproducible? Did you verify that the input to produce() was 1+1?

Reply:
This was an error in my test - intellisense had typed `Unicode` for me rather than `UTF8`.